cloudfoundry-client/src/main/java/org/cloudfoundry/client/v3/_Lifecycle.java

```diff
@@ -35,7 +32,11 @@
     /**
      * The datas
      */
-    @JsonDeserialize(using = DataDeserializer.class)
+    @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.EXTERNAL_PROPERTY, property = "type")
```

Comment:
@antechrestos I desperately want this to work, but I'm getting compilation errors with it.  Any chance you can double check compiling this with clean, from the command line?